index.html

```diff
@@ -357,14 +357,20 @@ <h2 class="billboard__header">Contact</h2>
     <script language="javascript" type="text/javascript">
       $(document).ready(function () {
         $('#Day-1').on('click', '.top', function () {
-          $(this).next().slideToggle();
-          $(this).find('.top-control .plus').toggleClass('hide');
-          $(this).find('.top-control .minus').toggleClass('hide');
+          var $this = $(this);
+          $(this).next().slideToggle(function () {
```

Comment:
Small nit to pick, though it shouldn't hold this back, why not use the variable you just declared `$this`?

Reply:
@claytron5000 : Yes, that's correct. I didn't notice that one. I have committed the change.